src/entities/document/components/DocumentNewForm/DocumentNewForm.tsx

```
@@ -0,0 +1,191 @@
+}: DocumentNewFormProps) => {
+    const titleId = useId();
+    const dropHintId = useId();
+
+    const {
+        register,
+        handleSubmit,
+        setValue,
+        watch,
+        getValues,
+        formState: { errors, isSubmitting: isFormSubmitting },
+    } = useForm<FormIn, any, FormOut>({
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)